middle_end/flambda2/from_lambda/closure_conversion_aux.ml

```diff
@@ -355,9 +418,10 @@ module Acc = struct
 
   let remove_continuation_from_free_names cont t =
     { t with
-      free_names = Name_occurrences.remove_continuation t.free_names cont;
-      continuation_applications =
-        Continuation.Map.remove cont t.continuation_applications
+      free_names =
+        Name_occurrences.remove_continuation t.free_names cont
+        (* continuation_applications =
+         *   Continuation.Map.remove cont t.continuation_applications *)
```

Comment:
You should replace this comment with a note explaining why the continuation is kept in `continuation_applications`